apps/io-services-cms-webapp/src/synchronizer/request-sync-cms-handler.ts

```diff
@@ -113,12 +113,6 @@ export const handleQueueItem = (
   pipe(
     queueItem,
     parseIncomingMessage(Queue.RequestSyncCmsItems),
-    E.mapLeft(
-      (err) =>
-        new Error(
-          `Error while parsing incoming message, the reason was => ${err.message}`,
-        ),
-    ), // TODO: map as _permanent_ error
```

Comment:
**question:** Why are these lines no longer useful?

Reply:
Last week we did a little refactor on queue handlers, the parseIncomingMessage was replicated in all the handler so we made a generic version, that you can find in io-services-cms/apps/io-services-cms-webapp/src/utils/queue-utils.ts, this mapLeft is a leftover from this refactoring